### ISSUE TITLE
nan returned in loss and val_loss keys

### ISSUE THREAD

**Issue by calag4n** [thibo73800/tensorflow2.0-examples#2]
Hi, 
When training the model in **Introduction - Simple Neural Network** file I've got :

```bash
Train on 8000 samples, validate on 2000 samples
Epoch 1/10
8000/8000 [==============================] - 1s 132us/sample - loss: nan - 
accuracy: 0.0934 - val_loss: nan - val_accuracy: 0.0975
[…]
```
returned by ```history = model.fit(images, targets, epochs=10, validation_split=0.2)```
So obviously nothing displayed by *plt* graphics.

I read the doc about the **fit** & **compile** methods but got no leads.

Running on :
Ubuntu 18.04
Python 3.8.1
conda 4.8.2
tensorflow-2.0.0

Nice work btw 👍️

**Comment by calag4n** [comment]
Well, 

After reinstalling anaconda, tensorflow and jupyter notebook in a fresh new environment, values come as expected. 
' Still don't know why though 😅️
